.github/CODEOWNERS

```
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in
+# the repo. 
+@nice-devone/DFO
```

I get reports that the `DFO` team doesn't exist. Maybe it would be better to select some other team.
```suggestion
@nice-devone/dfo-mobile-sdk-maintainers
```